README.rst

```diff
@@ -488,7 +494,7 @@ Execution
      -S, --suppress-metadata
                            Suppress metadata output.
      --metadata-output METADATA_OUTPUT
-                           File to which to write metadata records
+                           File to write metadata records
```

Comment:
This is actually generated by the helpstring in `arg_parsing`. Make sure you update it there too. https://github.com/awslabs/aws-encryption-sdk-cli/blob/master/src/aws_encryption_sdk_cli/internal/arg_parsing.py#L202-L206